lib/commons/dom/is-modal-open.js

```diff
@@ -10,7 +10,7 @@ import { querySelectorAllFilter } from '../../core/utils';
  * @instance
  * @return {Boolean|undefined} True if we know (or our best guess) that a modal is open, undefined if we can't tell (doesn't mean there isn't one open)
  */
-function isModalOpen(options) {
+export default memoize(function isModalOpen(options) {
```

Comment:
The options object is going to prevent memoize from working. This PR isn't about memoization anyway, so I don't think this belongs here. If we can't use the creator method for isModalOpen, best to just leave it as is.